src/components/Event/index.tsx

```diff
@@ -0,0 +1,71 @@
+import VolunteerList from '@/app/components/VolunteerList';
+import { EventResponse } from '@/types/dataModel/event';
+import { VolunteerResponse } from '@/types/dataModel/volunteer';
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+
+interface EventProps {
+  event: EventResponse;
+  volunteers: VolunteerResponse[];
+}
+
+export default function Event({
+  event,
+  volunteers,
+}: EventProps): React.ReactElement {
+  const eventNameLength = event.name.length;
+  const eventNameFontSize = eventNameLength > 20 ? 'h4' : 'h3';
+
+  // Format the date to a localized string
+  const formatDate = (date: Date): string => {
+    const option: Intl.DateTimeFormatOptions = {
+      weekday: 'long',
+      month: 'long',
+      day: 'numeric',
+    };
+    return date.toLocaleDateString(undefined, option);
+  };
+
+  // Format the time to a localized string
+  const formatTime = (date: Date): string => {
+    const option = {
+      hour: 'numeric' as const,
+      minute: 'numeric' as const,
+    };
+    return date.toLocaleTimeString(undefined, option);
+  };
+
+  // Format the end time to a localized string with time zone
+  const formatEndTime = (date: Date): string => {
+    const option = {
+      hour: 'numeric' as const,
+      minute: 'numeric' as const,
+      timeZoneName: 'short' as const,
+    };
+    return date.toLocaleTimeString(undefined, option);
+  };
```

Comment:
this app will only be used in EST, so we don't need to worry about displaying time zones

Reply:
alr ill take it out